jetstream/options.go

```diff
@@ -368,7 +389,8 @@ func (opt watchOptFn) configureWatcher(opts *watchOpts) error {
 	return opt(opts)
 }
 
-// IncludeHistory instructs the key watcher to include historical values as well.
+// IncludeHistory instructs the key watcher to include historical values as
```

Comment:
weird to see kv and object options here

Reply:
I'll move KV and object store options to separate files.

Reply:
Moved to separate files and renamed `options.go` to `jetstream_options.go`